app/code/community/Drip/Connect/Helper/Data.php

```diff
@@ -119,19 +119,35 @@ public function isSubscriberExists($email, $storeId = null)
     /**
      * check if customer exists on the website
      *
+     * @param string $email
+     * @param Drip_Connect_Model_Configuration $config
+     *
      * @return bool
      */
-    public function isCustomerExists($email, $websiteId = null)
+    public function isCustomerExists($email, Drip_Connect_Model_Configuration $config)
     {
-        if ($websiteId == null) {
-            $websiteId = Mage::app()->getStore()->getWebsiteId();
-        }
+        $websiteId = $config->getWebsiteId();
 
         $customer = Mage::getModel("customer/customer")->setWebsiteId($websiteId)->loadByEmail($email);
 
         return (bool) $customer->getId();
```

Comment:
It might make sense at this point to refactor this method into using `getCustomerByEmail`, since they are 90% identical.